frontend/Readme.md

```diff
@@ -41,15 +40,6 @@ go run main.go
 This will bring up the Service-Center UI on [http://127.0.0.1:30103](http://127.0.0.1:30103).
 If you want to change the listening ip/port, you can modify it in the configuration file (service-center/frontend/conf/app.conf : FRONTEND_HOST_IP, FRONTEND_HOST_PORT).
 
-##### NOTE
```

Comment:
How can we setup the proxy ? I can not find the document for it.

Reply:
The proxy is built into the frontend. And it's meant to be used by the JS api to access the Service Center apis. There are no need to configure/setup the proxy. It just starts up along with the frontend web service.